src/pydap/model.py

```diff
@@ -564,6 +567,16 @@ def __getitem__(self, key):
             return out
 
     @property
+    def output_grid(self):
+        if hasattr(self, '_output_grid'):
```

Comment:
It might simplify the code to ``set _output_grid()`` as a default value (e.g. True) in ``__init__()``. Then you don't have to check whether it exists... it will be safe to assume that it does.